core/request.go

```diff
@@ -175,5 +178,31 @@ func (r *RequestAccessor) ProxyEventToHTTPRequest(req events.APIGatewayProxyRequ
 	httpRequest.Header.Add(APIGwContextHeader, string(apiGwContext))
 	httpRequest.Header.Add(APIGwStageVarsHeader, string(stageVars))
 
+	lc, _ := lambdacontext.FromContext(ctx)
+	rc := requestContext{lambdaRuntime: lc, gatewayProxy: req.RequestContext}
+	ctx = context.WithValue(httpRequest.Context(), requestContextKey,
+		rc)
```

Comment:
This `rc` param should go on the previous line.

Reply:
Nice catch, it was due to refactor. Fixed.